### DISCUSSION TITLE
GitHub Action - Playwright test -> web server closes

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#190] [Q&A]
The web server Playwright launches keeps closing, but there's no hint as to why.

This is the error:
<img width="773" alt="Screenshot 2023-06-15 at 15 48 55" src="https://github.com/epicweb-dev/epic-stack/assets/22457463/d9caeea8-bff3-4c3a-a819-382c9e4d838a">

Playwright launches the web server successfully, as evidenced by the `🔶 Mock server installed` log, but it closes when running the `./server-build/index.js` (copied over from the `/server/index.ts`, to which I made no changes). So I'm a bit stumped as to why this might be 🤔

Running `npm run test:e2e:run` locally (which runs `npm run build` and `cross-env NODE_ENV=production MOCKS=true tsx .`) doesn't reproduce the issue.

Any idea what this might be?

**Comment by kentcdodds** [comment]
There's probably some issue running the server and the error is being swallowed up. This solves it: https://github.com/epicweb-dev/epic-stack/pull/183

**Comment by andrecasal** [comment]
Great, thanks!

I added an env var to the `/app/utils/env.server.ts` but was missing it in the `.env.example` file.
